app/views/triage/_patient_triage_card.html.erb

```diff
@@ -5,6 +5,7 @@
     </h2>
 
     <%= form_for [session, triage], builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= hidden_field(:triage, :patient_id) %>
```

Comment:
Does `f.hidden_field` work, is it preferable?

Reply:
I got rid of this in the last change to accommodate update, found a way around it.